system/filesystem.py

```diff
@@ -63,6 +63,44 @@
 - filesystem: fstype=ext4 dev=/dev/sdb1 opts="-cc"
 '''
 
+def _get_dev_size(dev, module):
+    """ Return size in bytes of device. Returns int """
+    blockdev_cmd = module.get_bin_path("blockdev", required=True)
+    rc, devsize_in_bytes, err = module.run_command("%s %s %s" % (blockdev_cmd, "--getsize64", dev))
+    return int(devsize_in_bytes)
+
+
+def _get_fs_size(fssize_cmd, dev, module):
+        """ Return size in bytes of filesystem on device. Returns int """
+        fssize_cmd = module.get_bin_path(fssize_cmd, required=True)
```

Comment:
If we assign full binary path to a new variable, for example `cmd`, we can later check for exact match against fssize_cmd, like `if 'tune2fs' == fssize_cmd`, and avoid useless `in` check.